Automattic/wp-calypso
# Add redirect to /log-in page when user is already logged in for WCCOM login flow

Related to #94047

## Proposed Changes

* Adds a check for logged-in user from woocommerce.com to automatically redirect to `/log-in` page instead of using wpcc page when rendering "Is this you" page

## Why are these changes being made?

* The page in wpcc is not well designed for woocommerce.com login, we should reuse the one from `/log-in`

#### Before

<img width="1077" alt="image" src="https://github.com/user-attachments/assets/c0c261c7-b135-4c53-aea8-06395d4ff1a5">

#### After

<img width="1079" alt="image" src="https://github.com/user-attachments/assets/12537f63-8ba6-4261-a737-760cd3ce1125">


## Testing Instructions

1. In an incognito window, go to https://woocommerce.com
1. Click on `Create an account`
2. Open another tab (same incognito window, go to https://wordpress.com/log-in/)
3. Log in using an existing account
4. Open the previous tab and refresh
5. Observe you're redirected to the `/log-in` auth page with `Continue as ...` button

## Pre-merge Checklist
